File: Programmers/src/Javascript/k의개수.js
function solution(i, j, k) {
    var answer = 0;
    let arr = [];
    let str = "";
    
    for(let z=i; z<=j; z++)
        arr.push(z);
    
    str = arr.join('');
        
    for(let n=0; n<str.length; n++) {
        if((str.charAt(n) - '0') == k)
            answer++;
    }
    
    
    return answer;
}

File: Programmers/src/Java/Lv0/k의 개수.java
package Java.Lv0;

class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;
        
        for(int a=i; a<=j; a++){
            StringBuilder sb = new StringBuilder();
            sb.append(a);
            int length = sb.length();
            
            for(int b=0; b<length; b++) {
                if((sb.charAt(b)-'0') == k)
                    answer++;
            }
        }
        
        return answer;
    }
}